packages/piro/test/_input_Analysis_ROL_AdjointSensitivities_OldReducedSpace_Tpetra.xml

```diff
@@ -151,8 +151,6 @@
         <Parameter name="Error File" type="string" value="dak.err" />
       </ParameterList>
       <ParameterList name="ROL">
```

Comment:
We should probably remove or rename (if it does something different than the other tests) this file

Reply:
I think that this test is redundant with one other test now.
I have to check that.